src/include/stir/deprecated.h

```diff
@@ -0,0 +1,39 @@
+/*
+    Copyright (C) 2000 PARAPET partners
+    Copyright (C) 2000- 2007, Hammersmith Imanet Ltd
+    Copyright (C) 2018-2019, University College London
```

Comment:
somewhat tricky as we got it from the internet. I'd just delete them, but add STFC of course